1/alpine/Dockerfile

```diff
@@ -60,7 +57,16 @@ RUN set -eux; \
 		su-exec node yarn add "sqlite3@$sqlite3Version" --force --build-from-source; \
 		\
 		apk del --no-network .build-deps; \
-	fi
+	fi; \
+	\
+	su-exec node yarn cache clean; \
+	su-exec node npm cache clean --force; \
+	npm cache clean --force; \
+	rm -r /tmp/*
```

Comment:
Is it possible for this to be more specific on what's getting deleted?  Do the files it leaves behind have some common prefix we could check for?

Reply:
Indeed, `/tmp/v8*` and `/tmp/yarn*` :+1: